Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -278,6 +287,13 @@ public struct __CommandLineArguments_v0: Sendable {
 
   /// The value of the `--experimental-attachments-path` argument.
   public var experimentalAttachmentsPath: String?
+
+  /// Whether or not the experimental warning issue severity feature should be
+  /// enabled.
+  ///
+  /// This property is intended for use in testing the testing library itself.
+  /// It is not parsed as a command-line argument.
+  var isWarningIssueRecordedEventEnabled: Bool?
```

Comment:
We can make this `public` the same way `public var experimentalAttachmentsPath: String?` is in the line above; the name should include "experimental" to reflect that it's not a finalized feature yet.

Actually, it should probably be `@_spi(Experimental)`, as should `experimentalAttachmentsPath`.

Reply:
I realize I might be giving conflicting advice here.

_If_ this is something we want to be able to set from the command line, we _should_ name it `experimentalWhatever` and include the ability to decode it so that it can be passed in a JSON blob.

If it is _only_ for testing purposes, then it's fine as-is.

Reply:
Right now, I don't need or want this property to be controllable outside the project, so I think it's appropriate as-is. I expect to formalize some mechanism to opt-in in a subsequent PR.